### PR TITLE
Documentation: Python Examples for Key Pair Creation

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

Added examples of how to create key pair (Ursa and no-Ursa) using python lib. 

### Benefits

People will know how to create keys

### Possible Drawbacks 

¯\_(ツ)_/¯

### CODE REVIEW

File: docs/source/getting_started/python-guide.rst [hunk 1]
```diff
@@ -13,7 +13,46 @@ To use Iroha Python library, you need to get it from the
...
+	import iroha.ed25519_sha2 as ed25519
+
+	private_key, _ = ed25519.create_keypair()
+	public_key = IrohaCrypto.derive_public_key(private_key)
+
+	with open('keypair.priv', 'wb') as f:
+	    f.write(private_key)
+
+	with open('keypair.pub', 'wb') as f:
+	    f.write(public_key)
```
Comment (MBoldyrev) [inline docs/source/getting_started/python-guide.rst:52]: with respect to [5c1c0f4](https://github.com/hyperledger/iroha-python/pull/54/commits/5c1c0f436b91a433e53a710fbbf24b3698b015c3),

```suggestion
	import binascii

	private_key = SigningKey.generate()
	public_key = private_key.verify_key
	
	with open('keypair.priv', 'wb') as f:
	    f.write(binascii.hexlify(private_key.encode()))
	    
	with open('keypair.pub', 'wb') as f:
	    f.write(binascii.hexlify(public_key.encode()))
```

File: docs/source/getting_started/python-guide.rst [hunk 2]
```diff
@@ -13,7 +13,46 @@ To use Iroha Python library, you need to get it from the
...
+
+And for HL Ursa ed25519 SHA-2 keys in Multihash format, please use:
+
+.. code-block:: python
+
+	from iroha import IrohaCrypto
```
Comment (MBoldyrev) [inline docs/source/getting_started/python-guide.rst:42]: also iroha lib is not required to generate ed25519 keypairs.
```suggestion
	import nacl.signing as ed25519_sha2
```

**Comment by LiraLemur** [comment]
All suggestions are committed @MBoldyrev

**Comment by MBoldyrev** [comment]
parts of this PR rely on not yet merged https://github.com/hyperledger/iroha-python/pull/54/ so although I am OK with the changes for now, something may change and we have to review it once the dependee is merged.

**Pull request merged**